app/controllers/users_controller.rb

```diff
@@ -1,11 +1,20 @@
 class UsersController < ApplicationController
   layout 'layouts/user_profile_layout', only: [:show]
 
+  skip_before_filter :verify_authenticity_token, :only => [:index, :show]
+
   before_filter :get_user, only: [:show, :add_status]
   before_filter :authenticate_user!, only: [:add_status]
 
   def index
-    @users = User.search(params)
+    @users = User.filter(set_filter_params).allow_to_display.by_create
+    @users_count = User.allow_to_display.count
+    @projects = Project.all
+
+    respond_to do |format|
+      format.js
+      format.html
+    end
```

Comment:
hmm… seem to be getting an error response when I use the “Close To My Timezone Area” option:

```
AbstractController::DoubleRenderError at /users
===============================================

> Render and/or redirect were called multiple times in this action. Please note that you may only call render OR redirect, and at most once per action. Also note that neither redirect nor render terminate execution of the action, so if you want to exit an action after redirecting, you need to do something like "redirect_to(...) and return".

app/controllers/users_controller.rb, line 14
--------------------------------------------

    9     def index
   10       @users = User.filter(set_filter_params).allow_to_display.by_create
   11       @users_count = User.allow_to_display.count
   12       @projects = Project.all
   13   
>  14       respond_to do |format|
   15         format.js
   16         format.html
   17       end
   18     end
   19   

App backtrace
-------------

 - app/controllers/users_controller.rb:14:in `index'
```

could you look into this?